library/xml-model/src/main/java/org/cru/godtools/xml/model/Text.kt

```diff
@@ -65,6 +76,9 @@ class Text : Content {
         _textColor = parser.getAttributeValueAsColorOrNull(XML_TEXT_COLOR)
         _textScale = parser.getAttributeValue(null, XML_TEXT_SCALE)?.toDoubleOrNull()
 
+        imageStartName = parser.getAttributeValue(null, XML_RESOURCE)
+        imageEndName = parser.getAttributeValue(null, XML_RESOURCE)
```

Comment:
you'll want to use the corresponding attribute names with this methods

Reply:
Aahhhh, gotcha! So this const is just for referencing the specific attribute you are trying to parse and will change depending on the attribute name.

Reply:
yeah, it's just extracting a string constant out